packages/plugin-react/src/index.ts

```diff
@@ -119,7 +119,7 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
         resolve: projectRoot
       })
       isProduction = config.isProduction
-      skipFastRefresh ||= isProduction || config.command === 'build'
+      skipFastRefresh = isProduction || config.command === 'build'
```

Comment:
I think it's a logical or assignment operator so the syntax is valid.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment

Reply:
But ts compiled that's not the case https://www.typescriptlang.org/play?ssl=12&ssc=22&pln=10&pc=1#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCbvCwDKgU8JkY7p7ehCTkVDQS2E6gnPCxGcwmZqDSTgzxxWWVoASMFmgYkAAeRJTInN3ymj4d-jSCeNsMq-wuoPaOltigAKoASgAywhK7SbGQZIIz5VWCFzSeCrZagNYbChbHaxUDcCjJZLfSDbExIAgUdxkUBIursJzCFJtXydajBdw0cQAXhxDGggkgAG5guIAD6smkARlAAD5QAAmUDs2n0pnBRQlIA

Reply:
There is no errors with the playground you sent. Does it show errors in your environment?
Also logical assignment operators are supported in TS 4.0+.
https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#short-circuiting-assignment-operators

Reply:
I'm sorry I misunderstood and I turned off the PR

Reply:
I agree with @sapphi-red too. It seems that the current code is correct, we don't have to remove `||=`